kratos/tests/test_element.cpp

```diff
@@ -0,0 +1,433 @@
+//    |  /           |
```

Comment:
also check the reactions if you may

Reply:
If I don't fix anything I don't have reaction, what do you suggest?